dom0-updates/qubes-dom0-update

```diff
@@ -66,13 +76,12 @@ done
 # Prevent implicit update of template - this would override user changes -
 # but do allow explicit template upgrade, downgrade, reinstall
 if [ "$YUM_ACTION" == "reinstall" ] || [ "$YUM_ACTION" == "upgrade" ] || [ "$YUM_ACTION" == "upgrade-to" ] \
-|| [ "$YUM_ACTION" == "downgrade" ] && [[ "$PKGS" == *"qubes-template-"* ]]; then
-    TEMPLATE_EXCLUDE_OPTS=""
+|| [ "$YUM_ACTION" == "downgrade" ] && echo "${PKGS[*]}" | grep -q '\<qubes-template-'; then
```

Comment:
`echo "${PKGS[*]}" | grep -q '\<qubes-template-'` will incorrectly assume a package such as `hello-qubes-template-debian-10` is a template... This will be fixed in a moment as well.

I am sorry for these multiple issues in what is supposed to be a simple change.